src/inputs/Table/components/TableToolbar/TableToolbar.spec.js

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import React from 'react';
+import { TableToolbar } from './TableToolbar';
+import { renderInMuiThemeProvider } from '../../../../../tests/utils';
+import { StackContainerTesting } from '../../../../../tests/MuiComponentsTesting';
+
+const tableToolbarInputContainer = new StackContainerTesting({ dataCy: 'table-toolbar' });
+const FullscreenButtonContainer = new StackContainerTesting({ dataCy: 'grid-fullscreen-button' });
+const ImportButtonContainer = new StackContainerTesting({ dataCy: 'import-file-button' });
+const ExportButtonContainer = new StackContainerTesting({ dataCy: 'export-button' });
+const AddRowButtonContainer = new StackContainerTesting({ dataCy: 'add-row-button' });
+const DeleteRowButtonContainer = new StackContainerTesting({ dataCy: 'delete-rows-button' });
+
+const defaultProps = {
+  isFullscreen: false,
+  toggleFullscreen: () => {},
+  isReady: false,
+  editMode: true,
+};
+const setUp = (props) => {
+  renderInMuiThemeProvider(<TableToolbar {...props} />);
+};
+
+describe('Check if TableToolbar buttons is displayed or not', () => {
+  test('Check if tableToolbar is displayed with default data', () => {
+    setUp(defaultProps);
+    expect(tableToolbarInputContainer.Stack).toBeInTheDocument();
+  });
```

Comment:
This test alone is not very interesting, unless there is a behavior that causes the toolbar not to exist in the DOM. If this is the case, we should add a test to check it is not in the DOM when these conditions are met :slightly_smiling_face: 

Also, several tests are setting up the component with the `defaultProps` and performing a single check: we could regroup these checks in a single `test()` case.

Reply:
DONE